src/Quartz.Benchmark/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
+
+  <PropertyGroup>
+    <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
```

Comment:
I guess setting directly in csproj would be easier?

Reply:
Yeah, I don't see why not. I'll change this in my next PR (unless you want a isolated PR for this).

Reply:
No worries, can be part of some future pr